tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,56 @@
-import pytest
-import SCF
-
-
-def test_calc_nuclear_repulsion_energy(mol_h2o):
-    assert True
-#    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077,\
-#        "Nuclear Repulsion Energy Test (H2O) Failed"
-
-
-def test_calc_initial_density(mol_h2o):
-    """
-    Tests that the initial density returns a zero matrix
-    and tests dimensions
-    """
-
-    Duv = SCF.calc_initial_density(mol_h2o)
-    assert Duv.sum() == 0.0
-    assert Duv.shape == (mol_h2o.nao,mol_h2o.nao)
+import pytest
+import SCF
+import main
+import pickle
+
+
+def test_calc_nuclear_repulsion_energy(mol_h2o):
+    assert SCF.calc_nuclear_repulsion_energy(mol_h2o) == 8.00236706181077, "Nuclear Repulsion Energy Test (H2O) Failed"
+
+
+def test_calc_initial_density(mol_h2o):
+    Duv = SCF.calc_initial_density(mol_h2o)
+    assert Duv.sum() == 0.0
+    assert Duv.shape == (mol_h2o.nao, mol_h2o.nao)
+
+
+def test_calc_hcore_matrix():
+    Tuv = pickle.load(open("tuv.pkl", "rb"))
+    Vuv = pickle.load(open("vuv.pkl", "rb"))
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    assert Huv[0, 0] == -32.57739541261037
+    assert Huv[3, 4] == 0.0
+
+
+def test_calc_fock_matrix(mol_h2o):
+    Tuv = pickle.load(open("tuv.pkl", "rb"))
+    Vuv = pickle.load(open("vuv.pkl", "rb"))
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    eri = pickle.load(open("eri.pkl", "rb"))
+    Duv = SCF.calc_initial_density(mol_h2o)
+    assert SCF.calc_fock_matrix(mol_h2o, Huv, eri, Duv)[0, 0] == -32.57739541261037, "Fock matrix test failed"
+
+
+def test_Roothan_equations_total_energy(mol_h2o):
+    Tuv = pickle.load(open("tuv.pkl", "rb"))
+    Vuv = pickle.load(open("vuv.pkl", "rb"))
+    Suv = pickle.load(open("suv.pkl", "rb"))
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    eri = pickle.load(open("eri.pkl", "rb"))
+    Duv = SCF.calc_initial_density(mol_h2o)
+    Fuv = SCF.calc_fock_matrix(mol_h2o, Huv, eri, Duv)
+    Enuc_ = SCF.calc_nuclear_repulsion_energy(mol_h2o)
+    mo_energies, mo_coeffs = SCF.solve_Roothan_equations(Fuv, Suv)
+    assert mo_coeffs[0, :] == pytest.approx([-1.00154358e+00, -2.33624458e-01,
```

Comment:
This is causing TravisCI to fail as stated in class and in the issue board, the signs of the values are not guaranteed from run to run and therefore one must test on absolute values.